### ISSUE TITLE
BadImageFormatException when running generate-load-scripts

### ISSUE THREAD

**Issue by mvkra** [fsprojects/Paket#2934]
### Description

When running paket.exe generate-load-scripts I get a failure of:

```
Paket failed with
-> BadImageFormatException: Format of the executable (.exe) or library (.dll) isinvalid.
```

I have traced the issue (by putting in printfn logging inside DependencyCache.fs - getDllsWithinPackage) to:

`Reading Assembly /home/user/.nuget/packages/microsoft.testplatform.testhost/15.5.0/lib/netstandard1.5/x64/msdia140.dll`

This is coming from the Microsoft.NET.Test.Sdk package. Note I am on Mono 5.4.0.

### Repro steps

Please provide the steps required to reproduce the problem

1. Bring in a solution that uses the "Microsoft.NET.Test.Sdk" package.
2. Attempt to run "mono .paket/paket.exe generate-load-scripts"

### Expected behavior

Load scripts are generated.

### Actual behavior

```
Paket failed with
-> BadImageFormatException: Format of the executable (.exe) or library (.dll) isinvalid.
```

### Known workarounds

None

**Comment by enricosada** [comment]
make sense the expection, that `x64/msdia140.dll` is a native library, not an assembly.

proposed fix? ignore that dll?

**Comment by mvkra** [comment]
Yep that's what I thought too since it is using Mono.Cecil.AssemblyDefinition to try and read the file which I assume expects IL code. I'm happy to ignore native DLL's for this case as right now I don't need it for FSI scripting. I don't think it is easy to deal with the native DLL problem anyway since they are loaded by the FSI process differently than the usual #r syntax (i.e it looks for them on the system PATH variable if I remember).
